hubverse_annotator/app.py

```diff
@@ -231,20 +231,66 @@ def reference_date_and_location_ui(
         the two letter location abbreviation.
     """
     loc_lookup = get_available_locations(observed_data_table, forecast_table)
-    long_names = loc_lookup["long_name"].to_list()
-    ref_dates = get_reference_dates(forecast_table)
+    if "locations_list" not in st.session_state:
+        st.session_state.locations_list = (
+            loc_lookup.get_column("long_name").sort().to_list()
+        )
+    if "location_selection" not in st.session_state:
+        st.session_state.location_selection = st.session_state.locations_list[
+            0
+        ]
+
+    def prev_loc():
```

Comment:
Maybe `go_to_prev_loc`, since this function actually changes the state? Similar suggestion for `go_to_prev_loc`?